pkg/loop/plugin_relayer.go

```diff
@@ -64,5 +63,8 @@ func (p *GRPCPluginRelayer) ClientConfig() *plugin.ClientConfig {
 		HandshakeConfig: PluginRelayerHandshakeConfig(),
 		Plugins:         map[string]plugin.Plugin{PluginRelayerName: p},
 	}
-	return ManagedGRPCClientConfig(c, p.BrokerConfig)
```

Comment:
Passing `p.BrokerConfig` here bypasses the service hierarchy. We need the `p.pluginClient.BrokerConfig`, which wasn't available yet, since it depended on the broker and connect as well. Now it can be constructed without them.